provider/cmd/pulumi-resource-nuage/nuage_provider/container_function.py

```diff
@@ -143,20 +176,21 @@ def __init__(self, name: str, args: ContainerFunctionArgs, props: Optional[dict]
             build = docker.DockerBuild(
                 context="./",
                 dockerfile="./Dockerfile.awslambda.generated",
```

Comment:
Perhaps we should use tempfile here ?
https://docs.python.org/3/library/tempfile.html